docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -33,16 +35,24 @@ width={1737}
 height={335}
 />
 
-#### Backfill policies and single-run backfills (Experimental)
+### Launching single-run backfills using backfill policies <Experimental />
+
+By default, if you launch a backfill that covers `N` partitions, Dagster will launch `N` separate runs, one for each partition. This works well when code is single-threaded, as it avoids overwhelming it with large amounts of data.
```

Comment:
on second read through, the line "This works well when code is single-threaded, as it avoids overwhelming it with large amounts of data." stands out as confusing to me. I'm also not really sure what it's trying to say

Reply:
I think this is from an update @sryza did recently - wanna weigh in here, Sandy?

My suggestion, though I'm not sure it's what you're going for:

> This approach can help avoid overwhelming Dagster or resources with large amounts of data.

Reply:
> However, if you're using a parallel-processing engine like Spark and Snowflake, you often don't need Dagster to help with parallelism, so splitting up the backfill into multiple runs just adds extra overhead.

Is taking this sentence out intentional? I think it helps add color to this.

Reply:
Nope - it probably happened when I was addressing the merge conflict that came up. Happy to add it back.